zorg/buildbot/builders/ClangBuilder.py

```diff
@@ -294,6 +307,13 @@ def _getClangCMakeBuildFactory(
             src_dir='test/test-suite',
             alwaysUseLatest=True)
 
+    if checkout_zorg:
+        f.addGetSourcecodeForProject(
+            project='zorg',
+            src_dir='llvm-zorg',
+            alwaysUseLatest=True,
+            repourl="https://github.com/asb/llvm-zorg.git")
```

Comment:
Needs to be the upstream URL.

Reply:
Well technically it needs to be the user's fork in some cases but hopefully the fact that nothing is changing will clue them into that :)

Reply:
Oops, that was a holdover from my local testing. Removed, so it default to upstream llvm-zorg. Thanks!